interceptor/pkg/interceptor/pdinterceptor_test.go

```
@@ -2,7 +2,7 @@ package interceptor
 
 import (
 	"errors"
-	"os"
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)